utils/winpackage.cmd

```diff
@@ -14,7 +14,9 @@ set ZipFile=azad.zip
 cd "%~dp0.."
 
 ::  run lint
-node node_modules\eslint\bin\eslint.js src/js
+::::::: Lint isn't working in Windows.
+:: tsc c:/Users/kille/Documents/GitHub/azad/src/js/*
+:::::::::::::::::::: node node_modules\eslint\bin\eslint.js src/js
```

Comment:
I'm going to remove eslint, because there's no more js source.
Instead "npm run lint" calls "tsc --noEmit src/js/*.ts"
I'd recommend other scripts call npm run lint.

Reply:
That's cheating! But npm run lint doesn't work for me anyway, even with full path. :-(

Reply:
we should figure that out... is tsc in your path? - I can't remember how I installed it - probably with npm.
Update: ```npm install --save-dev typescript ts-loader``` was the culprit.

Reply:
I ran that command and now I get a different error:

error TS6231: Could not resolve the path 'src/js/*' with the extensions: '.ts', '.tsx', '.d.ts'.